src/Nethermind/Nethermind.Api/Extensions/INethermindPlugin.cs

```diff
@@ -23,4 +23,5 @@ void InitTxTypesAndRlpDecoders(INethermindApi api) { }
     Task InitRpcModules() => Task.CompletedTask;
 
     bool MustInitialize => false;
+    bool Enabled { get; }
```

Comment:
Should it be `true` by default?

Reply:
I'd say no. Plugin should explicitly specify their condition.